src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let count = 0;
+
+  if (days < 3) {
+    count = days * 40;
+  } else if (days >= 3 && days < 7) {
+    count = (days * 40) - 20;
+  } else if (days >= 7) {
+    count = (days * 40) - 50;
+  }
```

Comment:
20, 40, and 50 are magic numbers
create variables for them for more readable code =)
![image](https://user-images.githubusercontent.com/105542865/189681055-a5ed96dc-d29d-4144-892e-130e062b856b.png)